src/QualificationRunner.Core/Constants.cs

```diff
@@ -13,6 +13,7 @@ public static class Constants
       public const string DEFAULT_REPORT_CONFIGURATION_PLAN_NAME = "report-configuration-plan";
       public const string CONTENT_FOLDER = "Content";
       public const string OBSERVED_DATA_FOLDER = "ObservedData";
+      public const string INPUTS_FOLDER = "Inputs";

```

Comment:
Convention: Where the input stuff will be created. This is pass along to the PKSim CLI which is going to dump stuff there